### PR TITLE
Manage Users

### PR DESCRIPTION
**Pull request by osimuka**

### CODE REVIEW

File: trade_remedies_public/templates/v2/manage_users/main.html [hunk 1]
```diff
@@ -0,0 +1,325 @@
+{% extends "v2/govuk/base.html" %}
+{% load alert_message %}
+{% load to_json %}
```
Comment (chris-pettinga) [inline trade_remedies_public/templates/v2/manage_users/main.html:3]: I think you can remove some of these templatetags as they are not used

**Review by chris-pettinga — COMMENTED** [review]
GitHub is bugging when I try and view the differences in application.css but I presume it's an indentation issue, do you mind aligning it with the original file?

File: trade_remedies_public/templates/v2/manage_users/main.html (hunk 1 above)
Comment (osimuka) [reply trade_remedies_public/templates/v2/manage_users/main.html:3]: @chris-pettinga good point, will fix now

File: trade_remedies_public/templates/static/v2/css/govuk/application.css [hunk 2]
```diff
@@ -1,450 +1,938 @@
 @charset "UTF-8";
-.app-step-nav-related__link, .app-step-nav-header__title, a, .govuk-link {
+.govuk-prototype-kit-manage-prototype-navigation {
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/static/v2/css/govuk/application.css:2]: Why do we have a bunch of prototyping kit CSS classes here?

Comment (osimuka) [reply trade_remedies_public/templates/static/v2/css/govuk/application.css:2]: Thats what we use to style it, its the design css, the application.css file is the file which most of the V2 uses @chris-pettinga, I'm I right in saying this

Comment (yusufsheiqh) [reply trade_remedies_public/templates/static/v2/css/govuk/application.css:2]: The prototyping kit is specifically used for design purposes.

> The GOV.UK Prototype Kit: is designed for prototyping, not for production code

Reference: https://prototype-kit.service.gov.uk/docs/about

File: requirements.in/base.in [hunk 3]
```diff
@@ -1,5 +1,5 @@
 git+https://github.com/uktrade/trade-remedies-client.git@master#egg=trade-remedies-client
-git+https://github.com/uktrade/trs_v2_api_client.git@master#egg=v2_api_client
+git+https://github.com/uktrade/trs_v2_api_client.git@feature/trsv2-460/manage-users#egg=v2_api_client
```
Comment (nahmed99) [inline requirements.in/base.in:2]: Is this correct?

File: requirements/base.txt [hunk 4]
```diff
@@ -155,7 +157,7 @@ urllib3==1.26.5
     #   botocore
     #   requests
     #   sentry-sdk
-v2-api-client @ git+https://github.com/uktrade/trs_v2_api_client.git@master
+v2-api-client @ git+https://github.com/uktrade/trs_v2_api_client.git@feature/trsv2-460/manage-users
```
Comment (nahmed99) [inline requirements/base.txt:160]: Same comment as above.

File: requirements.in/base.in (hunk 3 above)
Comment (yusufsheiqh) [reply requirements.in/base.in:2]: This is correct. Shall we merge the feature branch back in to the `master` branch?

File: trade_remedies_public/core/v2_forms/manage_users.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+class ChangeUserIsActiveForm(ValidationForm):
+    is_active = RequiredYesNoRadioButton()
+
+
+class ChangeCaseRoleForm(ValidationForm):
+    case_role = RequiredYesNoRadioButton()
+
+
+class AssignToCaseForm(ValidationForm):
+    ...
```
Comment (nahmed99) [inline trade_remedies_public/core/v2_forms/manage_users.py:45]: Perhaps a comment (docstring) to explain empty class?

Comment (yusufsheiqh) [reply trade_remedies_public/core/v2_forms/manage_users.py:45]: This might be redundant - good spot.

Comment (yusufsheiqh) [reply trade_remedies_public/core/v2_forms/manage_users.py]: Resolved in https://github.com/uktrade/trade-remedies-public/pull/265/commits/57d20209a42be3d14de9331aa9db01fb1a63bfd3.

**Review by nahmed99 — APPROVED** [review]

**Review by yusufsheiqh — APPROVED** [review]

**Pull request merged**